clickplc/driver.py

```diff
@@ -35,14 +37,25 @@ class ClickPLC(AsyncioModbusClient):
         'ds': 'int16',   # (D)ata register (s)ingle
         'dd': 'int32',   # (D)ata register, (d)ouble
         'dh': 'int16',   # (D)ata register, (h)ex
-        'df': 'float',   # (D)ata register (f)loating point
+        'df': 'float',   # (D)ata register (f)loating
+        'xd': 'int32',   # Input Register
+        'yd': 'int32',   # Output Register
         'td': 'int16',   # (T)imer register
         'ctd': 'int32',  # (C)oun(t)er Current values, (d)ouble
         'sd': 'int16',   # (S)ystem (D)ata register, single
         'txt': 'str',    # ASCII Text
     }
 
-    def __init__(self, address, tag_filepath='', timeout=1):
+    def __init__(self,
+                 address,
+                 tag_filepath='',
+                 timeout=1,
+                 interfacetype: Literal["TCP", "Serial"] = 'TCP',
+                 *,
+                 baudrate=38400,
+                 parity='O',
```

Comment:
`pyserial` has nice constants available for this purpose rather than using "raw" integers:
https://pyserial.readthedocs.io/en/latest/pyserial_api.html#serial.Serial